app/src/main/java/com/protify/protifyapp/features/events/EditEvent.kt

```diff
@@ -94,4 +101,31 @@ class EditEvent (private val eventToEdit: FirestoreEvent) : AddEvent() {
     fun EditEventPage(navigateBack: () -> Unit) {
         AddEventPage(navigateBack)
     }
+    private fun formatTime(startTime: LocalDateTime): String {
```

Comment:
Local date time needs to be formatted into a user readable string before it can be passed into the addevent page